eng/bomgenerator/generateAzureSDKBOM.ps1

```diff
@@ -14,6 +14,7 @@ $BomPomFilePath = Join-Path $RepoRoot "sdk" "boms" "azure-sdk-bom" $PomFileName
 $EngScriptDir = Join-Path  $EngDir "scripts"
 $BomGeneratorPomFilePath = Join-Path ${PSScriptRoot} $PomFileName
 $NewBomFilePath = Join-Path $OutputDir $PomFileName
+$GroupIds = @("com.azure", "com.azure.resourcemanager")
```

Comment:
Will we need to care about "com.azure.v2"? 

cc @alzimmermsft

Reply:
@weshaggard This PR doesn't touch core-v2. 

There's another potential artifactId duplication issue in core-v2's patch release, logged in: https://github.com/Azure/azure-sdk-for-java/issues/45768

Still, v2 patch release script may be an entirely different one.